### ISSUE TITLE
Button on the ErrorDisplay Component Violates WCAG Contrast Guidelines

### ISSUE THREAD

**Issue by sameersharmadev** [OWASP/Nest#2826]
**Describe the bug**
The "Return To Home" button in `ErrorDisplay` component in `/frontend/src/app/global-error.tsx` uses `text-white` on a light background (`bg-owasp-blue [#98AFC7]`), resulting in a poor contrast ratio (2.26:1), violating WCAG guidelines. Also, the background color does not adapt to dark mode.

**To Reproduce**
Visit any page that throws an error with the help of `ErrorDisplay` (eg: /projects/abc) and observe the "Return To Home" button.

**Suggested Fix**
Remove the hard-coded `text-white` class from the button. Add `dark:bg-slate-800` to improve dark-mode background. 

**Are you going to work on fixing this?**

- [x] Yes
- [ ] No

**Screenshots**
<img width="699" height="447" alt="Image" src="https://github.com/user-attachments/assets/76a7d328-9936-4424-a165-85b127b14608" />

**Desktop (please complete the following information):**

- OS: Linux
- Browser Chromium

**Comment by gsk0906** [comment]
I am willing to take this task and implement the suggested fix

**Comment by gsk0906** [comment]
I have a clear plan to fix this:

Remove the hard-coded text-white class.

Add dark:bg-slate-800 to support dark mode.

Ensure the text color dynamically adapts to maintain high contrast in both light and dark modes, fully complying with accessibility guidelines.

I’d like to take ownership of this fix and implement it promptly. This will improve both usability and accessibility, aligning with our commitment to inclusive design.

Please let me know if I can proceed—I’m confident this improvement will enhance the user experience significantly.

**Comment by kasya** [comment]
@sameersharmadev Thanks for creating this issue.
I've assigned this to you. Please also add/update tests for this page/button to cover changes.
